### PR TITLE
Powershell commandlet to enable diagnostics for V2 VM

### PR DESCRIPTION
**Pull request by rahulpandit85**
Contains following changes
1. Get, Set and Remove Azure Powershell commandlets for IaaS V2 VM.
2. Moved common code between ServiceManagement and ResourceManager diagnostics commandlets in to DiagnosticsHelper class.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@rahulpandit85__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci test this please

**Comment by hovsepm** [comment]
@rahulpandit85 , could you sign the CLA?

**Comment by rahulpandit85** [comment]
Hi,
I work for Microsoft Visual Studio Cloud tools team(rapandit@microsoft.com).
I don't think the CLA applies to me.
Let me know if otherwise.

File: src/ResourceManager/Compute/Commands.Compute/Commands.Compute.csproj [hunk 1]
```diff
@@ -121,6 +121,9 @@
       <HintPath>..\..\..\packages\Newtonsoft.Json.6.0.4\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.WindowsAzure.Management.Storage">
```
Comment (stankovski) [inline src/ResourceManager/Compute/Commands.Compute/Commands.Compute.csproj]: If this change adds a new DLL to the output please update WIX file

**Comment by stankovski** [comment]
Started on-demand job - http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-on-demand/37/
Started MSI build job - http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-msi/356/

**Comment by stankovski** [comment]
@rahulpandit85 merge conflicts

**Comment by rahulpandit85** [comment]
Merged with Azure\dev

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Pull request merged**